proto/controlclient.proto

```diff
@@ -37,3 +39,19 @@ message _ListCachesResponse {
   repeated _Cache cache = 1;
   string next_token = 2;
 }
+
+message _CreateSigningKeyRequest {
+}
+
+message _CreateSigningKeyResponse {
+  string key_id = 1;
+  bytes private_key = 2;
+  int32 created_at = 3;
```

Comment:
`created_at`? It's just an fyi for the customer that they could use to manage the key rotation.  During a previous meeting with Wyze we talked about they themselves managing the key rotation - whenever they want to rotate the key they'll generate a new one and delete the old one using the api.

Reply:
Yeah but isn't it possible for them to simply call `new Date()` or `System.currentTimeMillis()` or any of a hundred other "what time is it" functions? Why should we provide "current time" from this api?

Reply:
Yeah that's true.  I'll remove this but instead add `expires_at` (given that we will cap the ttl at a max of say 90 days).

Reply:
@wandaitzuchen - probably not related to the proto but more to dynamo where we will store the public key and id, we should store a `created_at` timestamp over there which we generate internally. This can help us in auditing or when we want to see a compromised private key generation.